manifests/foreman_proxy_content.pp

```diff
@@ -20,7 +20,7 @@
   $foreman_proxy_fqdn   = $certs::params::node_fqdn,
   $foreman_proxy_cname  = $certs::params::cname,
   $certs_tar            = $certs::params::certs_tar
-  ) inherits certs::params {
+) inherits certs {
```

Comment:
This class is meant to be used from a different context, without the certs class I believe.